runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/src/aws/smithy/kotlin/runtime/http/engine/okhttp/OkHttpEngine.kt

```diff
@@ -48,6 +50,8 @@ public class OkHttpEngine(
 
         val engineRequest = request.toOkHttpRequest(context, callContext, metrics)
         val engineCall = client.newCall(engineRequest)
+
+        @OptIn(ExperimentalCoroutinesApi::class)
         val engineResponse = mapOkHttpExceptions { engineCall.executeAsync() }
```

Comment:
From [OkHttp's changelog](https://github.com/square/okhttp/blob/master/CHANGELOG.md#version-500-alpha13):
> Breaking: Tag unstable new APIs as `@ExperimentalOkHttpApi`. We intend to release OkHttp 5.0 without stabilizing these new APIs first.
> 
> Do not use these experimental APIs in modules that may be executed using a version of OkHttp different from the version that the module was compiled with. Do not use them in published libraries. Do not use them if you aren't willing to track changes to them.

They're explicitly calling this out now as an unstable API which may change after 5.0 is fully released. We should carefully talk over the alternatives here.

Reply:
I think `ExperimentalOkHttpApi` is different from `ExperimentalCoroutinesApi`

Reply:
🤦‍♂️Oh you're right, I stopped reading at "Experimental". My mistake.